operations/accounts.md

```diff
@@ -62,3 +62,492 @@ Merges two or more accounts of the same account type together. The given source
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Accounts` | array of [Account](#account) | optional | Updated accounts. |
+
+#### Account
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the account. |
+| `Discriminator` | string [Account type discriminator](#account-type) | required | Type of the account. |
+| `Customer` | [Customer](#customer) | required | Updated customer data. |
+| `Company` | [Company](#company) | required | Updated company data. |
+
+#### Customer
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ChainId` | string | required | Unique identifier of the chain. |
+| `CreatedUtc` | string | required | Creation date and time of the customer in UTC timezone in ISO 8601 format. |
+| `UpdatedUtc` | string | required | Last update date and time of the customer in UTC timezone in ISO 8601 format. |
+| `UpdaterProfileId` | string | required | Unique identifier of the user who updated the customer. |
+| `Email` | string | optional | Email address of the customer. |
+| `TaxIdentifier` | string | optional | Tax identification number of the customer. |
+| `BillingCode` | string | optional | Billing code of the customer. |
+| `AccountingCode` | string | optional | Accounting code of the customer. |
+| `Notes` | string | optional | Internal notes about the customer. |
+| `LastName` | string | required | Last name of the customer. |
+| `FirstName` | string | optional | First name of the customer. |
+| `SecondLastName` | string | optional | Second last name of the customer. |
+| `BirthPlace` | string | optional | Place of birth. |
+| `Telephone` | string | optional | Telephone number of the customer \(possibly mobile\). |
+| `Occupation` | string | optional | Occupation of the customer. |
+| `CarRegistrationNumber` | string | optional | Registration number of the customer's car. |
+| `LoyaltyCode` | string | optional | Loyalty code of the customer. |
+| `NationalityCode` | string | optional | ISO 3166-1 code of the [Country](countries.md#country). |
+| `CompanyId` | string | optional | Unique identifier of [Company](companies.md#company) the customer is associated with. |
+| `BirthDate` | string | optional | Date of birth in ISO 8601 format. |
+| `Sex` | string [Sex](#sex) | optional | Sex of the customer. |
+| `Title` | string [Title](#title) | optional | Title prefix of the customer. |
+| `PreferredLanguageCode` | string | optional | Language and culture code of the customers preferred language. E.g. `en-US` or `fr-FR`. |
+| `Options` | [Customer options](#customer-options) | required | Options of the customer. |
+| `Classifications` | [Customer classifications](#customer-classifications) | required | Classifications of the customer. |
+| `LegalEntityIdentifiers` | [Legal entity identifiers](#customer-legal-identifiers) | required | Legal entity identifiers of the customer. |
+
+#### Company
```

Comment:
Yes, it uses different object. I don't think it should cause any issues for example in docs links etc. Maybe it could be little bit confusing, but to resolve that, we would have to name it differently - like Account company 🤔